front/tests/005-operational-studies.spec.ts

```diff
@@ -9,6 +9,14 @@ import { ProjectPage } from './pages/project-page-model';
 import { StudyPage } from './pages/study-page-model';
 
 test.describe('Testing if all mandatory elements simulation configuration are loaded in operationnal studies app', () => {
+  test('Testing there is no error', async ({ page }) => {
```

Comment:
Thanks for this test :pray:
However you put your test in the operational studies test file but this is not related to operational studies, can you move it to a new file, editor.spec.ts ? Thanks :)